src/otx/core/model/module/anomaly/anomaly_lightning.py

```diff
@@ -0,0 +1,450 @@
+"""Anomaly Lightning OTX model."""
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING, Any
+
+import onnx
+import openvino
+import torch
+from anomalib import TaskType
```

Comment:
would it be an idea to import this as AnomalibTaskType to avoid confusion later in the codebase